### PR TITLE
ENG-1519: Rebrands from RHODS to RHOAI and changes categories

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

File: docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml [hunk 1]
```diff
@@ -2,16 +2,16 @@ apiVersion: console.openshift.io/v1
 kind: QuickStarts
 metadata:
   name: openshift-ai-rhods-fasttrack
-  externalDocumentation: true
+  otherContentType: true
 spec:
   version: 0.1
   type:
-    text: Documentation
+    text: Other content type
```
Comment (bredamc) [inline docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml:9]: @Hyperkid123 This line is the other value that I'm not sure how to update. I don't know where to check for a list of accepted values.

File: docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml [hunk 2]
```diff
@@ -2,16 +2,16 @@ apiVersion: console.openshift.io/v1
...
     color: orange
-  displayName: FastTrack to Red Hat OpenShift Data Science
+  displayName: FastTrack to Red Hat OpenShift AI
   icon: >-
```
Comment (bredamc) [inline docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml:12]: @Hyperkid123 Should I remove the icon values, or specify them in some other way? The images are not shown on the cards in the UI?

**Review by bredamc — COMMENTED** [review]
@Hyperkid123 It's my PR, but some questions from me for you, please and thank you!

File: docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml (hunk 2 above)
Comment (Hyperkid123) [reply docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml:12]: You can just remove it. It was a UX decision not to show them anymore.

File: docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml (hunk 1 above)
Comment (Hyperkid123) [reply docs/quickstarts/openshift-ai-rhods-fasttrack/openshift-ai-rhods-fasttrack.yml:9]: Any string is accepted. Anything that can narrow down the document type would be helpful.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**